projects/gable/index.js

```diff
@@ -0,0 +1,18 @@
+const { get } = require('../helper/http')
+
+async function tvl(_, _b, _cb, { api, }) {
+  const response = await get('https://www.sundaefinance.net:8443/api/tvl_data');
```

Comment:
can you please fetch this info on chain? we have basic support for making calls to rdx chain: https://github.com/DefiLlama/DefiLlama-Adapters/blob/main/projects/helper/chain/radixdlt.js

Reply:
Hi @g1nt0ki, I have adjusted this accordingly.

The problem here is that the contract stores liquid staking units (LSU's). These LSU's are not tradable, they are redeemable for XRD on a 1:1 ratio. So essentially, the value of these LSU are equal to the value of XRD, but there is no price information available for this as they are not being traded.